File: word-count.kt
import  java.io.File
fun main(args : Array<String>) {
  File("911report.txt").readLines().map { 
    it.split(" ")
  }.flatten().groupBy { 
    it
  }.toList().map {  kv ->
    val (word, arr) = kv
    Pair(word, arr.size)
  }.sortedBy { 
    it.second * -1
  }.slice(0..20).map { 
    println( "${it.first} ${it.second}" )
  }
}  

File: word-count.rb

File.readlines('911report.txt').map { |x|
  x.split(' ').map { |x| x }
}.flatten.group_by { |x| 
  x 
}.to_a.map { |x| 
  key, arr = x
  [key, arr.size] 
}.sort_by { |x|
  x[1]*-1
}.slice(0..20).map { |x| 
  p x 
}
